recipes/dockermake/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "DockerMake" %}
+{% set version = "0.8.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7a1230ff1cd53e459a2ba9c3ea3d921a2b75d6907d300f41f8c9e5cabbb74c84
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+build:
+  number: 0
+  entry_points:
+    - docker-make = dockermake.__main__:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - docker-py >=3.2
+    - future
+    - python
+    - pyyaml
+    - termcolor
+
+test:
+  imports:
+    - dockermake
+  commands:
+    - docker-make --help
+
+about:
+  home: https://github.com/avirshup/dockermake
+  license: Apache-2.0
+  license_family: APACHE
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
This can be replaced with just LICENSE, cb3 has logic to find the recipe in the recipe folder or in the source folder.

Reply:
`LICENSE` is actually included in the tarball.  But because this is an APACHE license, I'm often confused if that means that I still need to include the `LICENSE` with the recipe of not.  Based on [the docs](https://conda-forge.org/docs/meta.html#packaging-the-license-manually), I think the answer is that as long as the `LICENSE` in the tarball, it doesn't need to be included with the recipe.

Reply:
If the license is in the source tarball you do not need to include another copy of it in the recipe directory. However, you still need the `license_file` entry in meta.yaml.